docs/prover/cli/options.md

```diff
@@ -215,6 +215,61 @@ When you have a rule with multiple assertions:
 **Example**
 `certoraRun Bank.sol --verify Bank:Bank.spec --multi_assert_check`
 
+(--independent_satisfy)=
+### `--independent_satisfy`
+
+**What does it do?**
+The independent satisfy mode checks each {ref}`satisfy statement <satisfy>` independently from all other satisfy statements that occurs in a rule. 
+Normally, each satisfy statement will be turned into a sub-rule (similarly to the {ref}`--multi_assert_check` mode), 
+but previously encountered satisfy statements will be still considered when creating a satisfying assignment.
+Turning on the `independent_satisfy` mode will ignore all currently unchecked satisfy statements for each sub-rule.
+
+As an illustrative example, consider the following rule `R` that has two satisfy statements:
+
+```cvl
+rule R {
+  bool b;
+  satisfy b, "R1";
+  satisfy !b, "R2";
+}
+```
+
+The statements for "R1" and "R2" will actually create two sub-rules equivalent to:
+```cvl
+rule R1_default {
+  bool b;
+  satisfy b, "R1";
+}
+
+rule R2_default {
+  bool b;
+  assume b, "R1";
+  satisfy !b, "R2"; // This actually checks b && !b
+}
+```
+
+The `independent_satisfy` mode would also generate and check two sub-rules: `R1` where `b` is satisfied (by `b=true`) while `satisfy !b` is removed, and `R2` where `satisfy b` is removed, and `!b` is satisfied (by `b=false`).
+Without turning `independent_satisfy` mode on, `R2` would have failed, as it would try to satisfy `b && !b`. The two `independent_satisfy` generated sub-rules will be equivalent to:
```

Comment:
I would start a new paragraph before "The two", because it's not clear that we're switching to the new example.

Reply:
🫡